.gitignore

```diff
@@ -48,3 +48,6 @@ lerna-debug.log*
 
 # Database mounted volume
 data
+
+# ABIs
+/src/abis/safe
```

Comment:
nitpick: newline a the end of file

Reply:
Inconsequential as of 3ee2bbeb0c6ad411221151d30b9ead8cc3331cef.